LK_proposals/LK Proposal for Collaboration.md

```diff
@@ -0,0 +1,116 @@
+## Proposal for Collaboration within our simulation structure
+
+There exists substantial flexibility in the conduction of adversarial collaboration (AC) (Mellers et al., 2001;  Cowan, 2020; Melloni et al., 2021). The below presented approach tries to incorporate suggestions from the literature, while also introducing deviances due to our use case.
+
+**First round: seperate studies:**
+
+In order to have a starting point of different perspectives, as well as to be able to investigate whether AC shows incremental improvements over preregistration, a preliminary simulation study should be pursued by both researchers on their own.
+
+<ins>Procedure
+
+1. 2 seperate preregistrations = simulation protocol Morris et al. (2019)
+    
+    —> remaining RDoF/QRPs in operationalization, hypothesis generation, method selection and interpretation
+    
+
+*Open question*: Should we Red Team on our opposites preregistration? Would mitigate Bias and would differ from AC, as one would not have to follow critique if it feels unwarrented
```

Comment:
However if we really want to have the proposed comparison between preregistration and increments of AC, this would need to be under disclosure until after the single studies have been conducted.

Reply:
I agree